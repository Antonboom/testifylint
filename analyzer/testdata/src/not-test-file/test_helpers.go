package service

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

func assertTrue(t *testing.T, predicate bool) {
	t.Helper()

	assert.Equal(t, true, predicate)                                   // want "bool-compare: use assert\\.True"
	assert.Equalf(t, true, predicate, "msg with args %d %s", 42, "42") // want "bool-compare: use assert\\.True"
}
