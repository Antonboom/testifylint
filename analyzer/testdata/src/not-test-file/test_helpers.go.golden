package service

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

func assertTrue(t *testing.T, predicate bool) {
	t.Helper()

	assert.True(t, predicate)                                   // want "bool-compare: use assert\\.True"
	assert.Truef(t, predicate, "msg with args %d %s", 42, "42") // want "bool-compare: use assert\\.True"
}
